docs/Course Wiki/ECE Course Offerings/ECE310.md

```diff
@@ -0,0 +1,79 @@
+
+This class does not require [ECE311](ECE311.md) (the lab section) but it is highly recommended to take it concurrently with ECE310. It is also recommended to take ECE310 soon after ECE210 as the courses do have some overlap as it pertains to transforms and system properties. Do not fret! Doing well in ECE210 is not required to do well in ECE310. 
+
+This course primarily deals with LTIC systems.
+
+## Prerequisites
+
+- [ECE210](ECE210.md)
+
+[ECE210](ECE210.md) is the only official prerequisite to the course. However, the course teaches students everything about systems, BIBO stability, Linearity, and Fourier transforms, so it is not necessary to remember any of these from ECE210. It is useful, not necessary, to know basic Linear Algebra before ECE310 to better understand the theory and arithmetic behind discrete convolution.
+
+## When to Take it
+
+Take this course if you have any sort of passing interest in controls. Control systems are also important in fields such as power systems and robotics. Be warned that the course does require a decent time commitment due to the amount of content and work required. If you do wish to take this course as soon as possible, take [ECE210](ECE210.md) as soon as possible.
+
+## Course Structure
+
+This course has weekly homeworks, two midterms, and one final.
+
+The homeworks start weekly and tend to lighten up as the semester goes on. On exam weeks, the homework due date is usually extended. Homework sets are generally not too difficult and will contain both calculations and derivations of expressions. Some homeworks will take under an hour, while others require more calculation and can be generally annoying. Students who pay attention and stay engaged in lecture will find these homeworks to be conceptually simple.
+
+The exams are usually pretty difficult, but less difficult than 210 exams for most people. The averages are usually around upper 60s to upper 70s with varying standard deviations. Very few practice exams are given out (1-3 per exam) and students are expected to collaborate to answer some of the practice exams as the answer sheets are sometimes not available. It is recommended to go to office hours to get the complete solution sets for all of the practice exams, only if you have tried the problems. Cheatsheet policies may vary depending on the semester and the lead instructor. For the Spring 2023 semester, students were allowed to bring one double-sided 8.5x11 cheatsheet to the first exam, two double-sided 8.5x11 cheatsheets to the second exam, and three double-sided 8.5x11 cheatsheets to the final exam. Students were highly recommended to reuse cheatsheets from prior exams, although this was not enforced.
```

Comment:
"It is recommended to go to office hours to get the complete solution sets for all of the practice exams, but only if you have tried the problems" (add "but")